components/questionCard.js

```diff
@@ -239,10 +266,7 @@ const QuestionCard = ({
           />
         </>
       )}
-      {isToggled &&
-      [QUESTION_TYPES.MCQ, QUESTION_TYPES.DROPDOWN, QUESTION_TYPES.SELECTALL, QUESTION_TYPES.CHECKBOX].includes(
-        question.type
-      ) ? (
+      {isToggled && [QUESTION_TYPES.MCQ, QUESTION_TYPES.DROPDOWN, QUESTION_TYPES.SELECTALL].includes(question.type) ? (
         <div>
           <QuestionTitle color={`${COLOR.MIDNIGHT_PURPLE_DEEP}`}>Options</QuestionTitle>
```

Comment:
if you're bored/have nothing to do then you could get rid of the `${}` in these LOL

Reply:
o github didn't highlight that correctly, meant to say `color={`${COLOR.MIDNIGHT_PURPLE_DEEP}`}` could just be`color={COLOR.MIDNIGHT_PURPLE_DEEP}`